pages/docs/v3/guides/environment-specific-configurations.md

```diff
@@ -0,0 +1,236 @@
+
+The Capacitor configuration file handles high-level options for Capacitor tooling and plugin configuration. iOS schemes and Android product flavors allow developers to provide differing app values for different environments. By combining the two, developers can use the Capacitor CLI to build apps for different environments.
+
+This guide will walk you through setting up a QA environment configuration alongside the default environment configuration provided out-of-the-box. To demonstrate differences between each environment, the app name and bundle ID will differ between the two.
+
+## Prepare a Capacitor App
+
+You will need a Capacitor app with both iOS and Android platforms added. If you have an existing Capacitor app with both platforms added, skip this section.
+
+Depending on your preference, you can either [add Capacitor to an existing web application](/docs/getting-started) or [create a new Capacitor application with the Ionic Framework](/docs/getting-started/with-ionic).
+
+The Capacitor app must use TypeScript for configuration. This guide uses `capacitor.config.ts` to dynamically export different configurations.
+
+Before adding any native platforms to the project, you must build the Capacitor App at least once.
+
+```bash
+npx run build
+```
+
+Once built, you can add the platforms.
+
+```bash
+npm install @capacitor/{ios,android}
+npx cap add ios
+npx cap add android
+```
+
+## Setup a new iOS scheme
+
+### Create a new Xcode target
+
+Start by opening the native iOS project in Xcode: `npx cap open ios`.
+
+1. Go to the project's settings in the Project Navigator panel. Under the _Targets_ section, right-click the "App" target and select **Duplicate** to copy the existing target.
+2. Click the new "App copy" target and press the `Enter` key to rename it. Set the target's name to "App QA".
+
+This process created an additional "App copy" scheme and added a new file called `App copy-Info.plist`.
+
+You can find additional information on iOS targets [at this link](https://developer.apple.com/library/archive/documentation/ToolsLanguages/Conceptual/Xcode_Overview/WorkingwithTargets.html).
+
+### Rename the new scheme and Plist file
+
+1. Select **Manage Schemes...** from the Scheme menu.
+2. Find the "App copy" scheme and press the `Enter` key to rename it. Set the name to "App QA" and close the dialog.
+3. Find the "App copy-Info" file in the Project Navigator panel and press the `Enter` key to rename it. Set the file's name to "App QA-Info.plist".
+4. Return to the project's settings. Ensuring that the "App QA" target is selected, open the _Build Settings_ section. Scroll down to Packaging and change the **Info.plist File** entry to "App QA-Info.plist".
+
+The iOS project now has two runnable schemes: "App" and "App QA". Capacitor's configuration file allows you to supply which scheme to build during the `open` and `run` commands.
```

Comment:
the scheme is not used on open command, open command just opens the workspace

Reply:
Removed, thanks for pointing this out!